setup.py

```diff
@@ -34,7 +34,7 @@
 description = "Cloud IoT API client library"
 version = "1.0.5"
 release_status = "Development Status :: 5 - Production/Stable"
-dependencies = ["httpx"]
+dependencies = ["httpx", "proto.datetime_helpers"]
```

Comment:
I'm not 100% sure if this would work. I think you'll need to add the exact pip package `proto-plus` in this case.

Reply:
OK, I'll make that change.